samplekotlin/src/main/java/com/example/chattutorial/ChannelActivity4.kt

```diff
@@ -86,7 +89,8 @@ class ChannelActivity4 : AppCompatActivity() {
         ChatClient
             .instance()
             .channel(cid)
-            .subscribeFor(this, TypingStartEvent::class.java, TypingStopEvent::class.java
+            .subscribeFor(
+                this, TypingStartEvent::class.java, TypingStopEvent::class.java
```

Comment:
We have extensions to make this play nicer with Kotlin, accepting `KClass` parameters:

```suggestion
                this, TypingStartEvent::class, TypingStopEvent::class
```